packages/web/src/assets/css/global.css

```
@@ -0,0 +1,12 @@
+/* Import GC Design System Tokens */
+@import '../../../../../node_modules/@cdssnc/gcds-tokens/build/web/css/tokens.css';
```

Could we remove the relative path and switch the import path to:
```suggestion
@import '~@cdssnc/gcds-tokens/build/web/css/tokens.css';
```